tools/library/tbc/lddecodemetadata.cpp

```diff
@@ -24,6 +24,25 @@
 
 #include "lddecodemetadata.h"
 
+// Default line parameters for PAL decoding
```

Comment:
Please could you keep the comments that were in getVideoParameters, explaining why these defaults were chosen?

Reply:
Addressed; comments re-added.